ditto/writers/opendss/write.py

```diff
@@ -2025,39 +2144,62 @@ def write_loads(self, model):
-                        ):
-                            filename = self.timeseries_datasets[
-                                substation + "_" + feeder
-                            ][ts.data_location]
-                            if self.remove_loadshapes:
-                                optional_comment = '!'
-                            else:
-                                optional_comment = ''
-                            txt += " {optional_comment}{ts_format}={filename}".format(
-                                optional_comment = optional_comment,
-                                ts_format=self.timeseries_format[filename],
-                                filename=filename,
-                            )
-                        else:
-                            pass
-                            # TODO: manage the data correctly when it is only in memory
+                    txt = append_time_series(i, txt)
 
                 txt += "\n\n"
-                feeder_text_map[substation_name + "_" + feeder_name] = txt
+
+            else:  # unbalanced load, need to define more than one New Load
+                for (n, phase_load) in enumerate(i.phase_loads):
+                    txt += "New Load." + i.name + "_" + str(n+1)  # need unique names
+                    txt += bus1_txt
+                    if hasattr(phase_load, "phase") and phase_load.phase is not None:
+                        dss_phase = self.phase_mapping(phase_load.phase)
+                        txt += f".{dss_phase}"
+
+                        # handle Delta connections
+                        if i.connection_type == "D":
+                            if dss_phase == 1:
+                                txt += ".2"
+                            if dss_phase == 2:
+                                txt += ".3"
+                            if dss_phase == 3:
+                                txt += ".1"
+
+                    txt += " Phases=1"
+
+                    nomimal_voltage_txt = ""
+                    if hasattr(i, "nominal_voltage") and i.nominal_voltage is not None:
+                        nominal_voltage = i.nominal_voltage * 10 ** -3
+                        if not (i.connection_type == "D" or nominal_voltage < 300):
+                            # single phase connections in openDSS should specify LN voltage
+                            # if not delta connection or not already LN
+                            nominal_voltage = round(nominal_voltage / math.sqrt(3), 4)
+                        nomimal_voltage_txt = f" kV={nominal_voltage}"
+
+                    txt += nomimal_voltage_txt
+
+                    txt += " kW={P}".format(P=kws[n])
+
+                    txt += " model={N}".format(N=phase_load.model)
+
+                    txt += conn_type_txt
+
+                    txt += " kvar={Q}".format(Q=kvars[n])
+                    
+                    txt += vmin_txt
+                    txt += vmax_txt
```

Comment:
See comment above. I think some aspects of the original mapping might not be included here. Is that intentional?
I think that this could be restructured to include duplication with the balanced section.